gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -437,6 +493,68 @@ public static ZoneFilter notEquals(ZoneField field, String value) {
   }
 
   /**
+   * Class for filtering operation lists.
+   */
+  class OperationFilter extends ListFilter {
+
+    private static final long serialVersionUID = -3202249202748346427L;
+
+    OperationFilter(OperationField field, ComparisonOperator operator, Object value) {
+      super(field.selector(), operator, value);
+    }
+
+    /**
+     * Returns an equality filter for the given field and string value. For string fields,
+     * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
+     * match the entire field.
+     *
+     * @see <a href="https://github.com/google/re2">RE2</a>
```

Comment:
I think https://github.com/google/re2/wiki/Syntax is slightly more useful than https://github.com/google/re2 in this context.